mkosi.conf.d/centos/10-centos.conf

```diff
@@ -102,6 +102,7 @@ BuildPackages=
         pkgconfig(tss2-rc)
         pkgconfig(valgrind)
         pkgconfig(xkbcommon)
+        python3-docutils
```

Comment:
Is it not necessary to follow the way how to specify module?
```
python3[.][9]dist(docutils)
```
I am not familiar with how we should specify the package, just a question.

Reply:
In this case, no, because we only use the scripts from the package, not the python modules. It's just unfortunately named the same as regular python modules, but it has a very different purpose.

Ideally this package would be named `docutils